packages/create-docusaurus/templates/shared/docs/intro.md

```diff
@@ -3,33 +3,54 @@ sidebar_position: 1
 ---
 
 # Tutorial Intro
+A beginners guide <br />
 
 Let's discover **Docusaurus in less than 5 minutes**.
 
 ## Getting Started
 
-Get started by **creating a new site**.
+Get started by easily **creating a new site**
 
 Or **try Docusaurus immediately** with **[docusaurus.new](https://docusaurus.new)**.
 
+### What you'll need
+
+- [Node.js](https://nodejs.org/en/download/) version 14 or above:
```

Comment:
This seems too long for a quick tutorial. Won't just an installation link + a sentence telling users to check all boxes to install all related dependencies suffice? How many users don't know how to install Node from a website's GUI?

I suggest to also include the changes in the "installation" page in the actual docs.